doc/programming_guide/event_loop.rst

```diff
@@ -5,34 +5,154 @@ Introduction
 ------------
 
 Python Arcade provides a simple three event loop to build off.
```

Comment:
Python Arcade provides three simple hook methods to integrate with the event loop.